PKPD/model/model.py

```diff
@@ -0,0 +1,83 @@
+from PKPD.model.abstractModel import AbstractModel
+import myokit
+
+class Model(AbstractModel):
+    def read_mmt_file(self):
+        # calls self.mmtfile to read
+        # sets variables: initial values, myokit model, protocol, parameters, units
+
+        #Set up Myokit Model
+        self._set_model()
+
+        # List of Initial States
+        self.initial_values = self.model.state()
+
+        #Name of Central Compartment
+        self.central_compartment_name = next(self.model.states()).qname()
+
+        #Number of States
+        self.dimension = len(self.initial_values)
+
+        #Dictionary of Parameters
+        self.parameter_component_name = 'param'
+        self.params = {var.qname(): var.eval() for var in self.model.get(self.parameter_component_name).variables()}
+
+    def set_mmt_file(self, filename):
+        self.mmtfile = filename
+
+    def set_initial_values(self, initial_values):
+        """
+        Sets initial values (state) in myokit model
+        :param initial_values: list
+        :return:
+        """
+        #Update in MyokitModel
+        self.model.set_state(initial_values)
+
+        #Update in Attributes
+        self.initial_values = self.model.state()
+
+    def _set_model(self):
+        self.model, self.protocol, _ = myokit.load(self.mmtfile)
+
+    def set_protocol(self, protocol):
+        """
+        :param protocol: myokit protocol object
+        :return:
+        """
+        self.protocol = protocol
+
+    def set_params(self, **params):
```

Comment:
I just realized that the we probably don't want to have `**params` here but rather just pass the dict as it is, i.e. `set_params(self, **params)` otherwise it breaks for multiple parameters passed at once.

Reply:
Then you would also no longer need the construction `self.parameter_component_name + '.' + name` and could just use `name`.